### PR TITLE
simple loading text and remove unused constants and unused case stmt in market info reducer.

### PR DESCRIPTION
**Pull request by bthaile**
add simple loading... text to orderbook when orderbook isn't fresh
remove unused property market-view, remove unused constants and unused case stmt in market info reducer.

add simple loading text. 
image

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://dashboard.featurepeek.com). 🔮
Your deployment will be kept up-to-date with this pull request's latest changes.

**https://peek.run/5m3hr9m**

-------

*Please read [our docs](https://docs.featurepeek.com/configuring-peek-yml/) for more configuration details.*

File: packages/augur-ui/src/modules/market/components/market-view/market-view.tsx [hunk 1]
```diff
@@ -504,6 +502,7 @@ export default class MarketView extends Component<
     }
 
     let outcomeOrderBook = this.EmptyOrderBook;
+    const orderbookLoading = !!!orderBook;
```
Comment (andrewdamelio) [inline packages/augur-ui/src/modules/market/components/market-view/market-view.tsx]: Looks like you may mean `!orderbook`

Comment (bthaile) [reply packages/augur-ui/src/modules/market/components/market-view/market-view.tsx]: it's an object, converting to boolean and checking if false.

Comment (bthaile) [reply packages/augur-ui/src/modules/market/components/market-view/market-view.tsx]: actually, could just use !orderBook. I'll make the change

**Comment by bthaile** [comment]
made modifications to PR to use `!orderBook` instead of `!!!orderBook`

**Review by andrewdamelio — APPROVED** [review]

**Pull request merged**